### PR TITLE
Fix build-image cronjob

### PR DESCRIPTION
**Pull request by cbartz**
### Overview

Fix the cron file by 

- adding a new line at the end
- using the absolute path of the build script

Also, add the `ubuntu` user to the `lxd` group to get the right permissions.

### Rationale

Scheduling the building of images does not work. `/var/log/syslog` contains

```
Feb 26 21:57:01 juju-e239d1-2 cron[483]: (*system*build-runner-image) ERROR (Missing newline before EOF, this crontab file will be ignored)
```

In addition, the `ubuntu` user does not have sufficient privileges to access the lxd socket.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: tests/unit/test_runner_manager.py [hunk 1]
```diff
@@ -56,8 +56,8 @@ def charm_state_fixture():
             GithubRepo("test_owner", "test_repo"),
             ProxyConfig(
                 no_proxy="test_no_proxy",
-                http_proxy=TEST_PROXY_SERVER_URL,
-                https_proxy=TEST_PROXY_SERVER_URL,
+                http=TEST_PROXY_SERVER_URL,
```
Comment (arturo-seijas) [inline tests/unit/test_runner_manager.py:59]: Why this change?

Comment (cbartz) [reply tests/unit/test_runner_manager.py:59]: It was wrong before, ProxyConfig no longer contains the `http_proxy' attrs.

**Review by arturo-seijas — APPROVED** [review]

**Review by javierdelapuente — APPROVED** [review]
